lib/voting.go

```diff
@@ -116,6 +116,7 @@ func NewVotingResult(ballot Ballot) (vr *VotingResult, err error) {
 		Source:      ballot.Source(),
 		State:       ballot.State(),
 		Ballots:     ballots,
+		Staging:     make([]VotingStateStaging, 0),
```

Comment:
Is this really necessary ? AFAIK it makes an empty slice ?

Reply:
Oh, IMHO, it is not :-)

Reply:
Oh, I see 👍